# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/fvwm/lto.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

LTO-specific duplicate symbol '_XA_WM_PROTOCOLS'.

--- fvwm-2.7.0/modules/FvwmScroll/GrabWindow.c.vanilla	2021-01-17 16:43:21.218000000 +0100
+++ fvwm-2.7.0/modules/FvwmScroll/GrabWindow.c	2021-01-17 16:43:44.573000000 +0100
@@ -80,7 +80,7 @@
 		     ButtonPressMask | ButtonMotionMask | FocusChangeMask)
 
 Atom wm_del_win;
-Atom _XA_WM_PROTOCOLS;
+static Atom _XA_WM_PROTOCOLS;
 Atom _XA_WM_COLORMAP_WINDOWS;
 
 /*
